api/lib/error-messages.js

```diff
@@ -2,19 +2,26 @@ module.exports.TYPE_MISMATCH_ERROR = function(parameter, expected, received) {
     return `Wrong type for parameter ${parameter}. Received ${received}, expected ${expected}`;
 }
 
-module.exports.MISSING_FIELD_ERROR = function(field){
+module.exports.MISSING_FIELD_ERROR = function(field) {
     return `Missing field "${field}"`;
 }
 
 module.exports.IO_ERROR = function(file){
     return `File read/write error when trying to access file ${file}`;
 }
 
-module.exports.DB_CONNECT_ERROR = () => `Error connecting to the database, check logs for details.`;
+module.exports.DB_CONNECT_ERROR = () => 'Error connecting to the database, check logs for details.';
+
 module.exports.NO_DB_ERROR = () => 'This client has no default databaset set, and one was not provided.';
 
-module.exports.LIMIT_FILE_SIZE_ERROR = function(maximumSize){
+module.exports.LIMIT_FILE_SIZE_ERROR = function(maximumSize) {
     return `The uploaded file exceeds the maximum file size supported of ${maximumSize/1024/1024}MB.`;
 }
 
+module.exports.INVALID_SCHEMA_ERROR = function(validSchemas) {
+    return `Schema must be ${validSchemas}`;
+}
 
+module.exports.GENERAL_SCHEMA_ERROR = function(generalError) {
```

Comment:
could you provide a usage example of these two new errors?

Reply:
Sure!

Reply:
```js

json.isValid({"restaurant":{ "name":"test", "address":"test", "city":"test"}}, 'men').then((document)=>{
    console.log(document)
})
.catch((err) => {
    console.log(err)
})
/*
    Error: Schema must be menu.
*/


json.isValid({"restaurant":{ "name":1223, "address":"test", "city":"test"}}, 'menu').then((document)=>{
    console.log(document)
})
.catch((err) => {
    console.log(err)
})
/*
    Invalid object: data.restaurant.name should be string
*/

```

Reply:
Oh that is not good :laughing:  I didn't thought that through... because I think these two errors are just gonna be used on the JSONValidator.js - and the GENERAL_SCHEMA_ERROR is even worse because it is just using the ajv.errorsText(validate.errors)... Do you think that is better take off these two error messages and just throw them without the 'error-messages' lib on the JSONValidator.js? like that:

Reply:
```js
		if (!this.validSchemas.includes(schema)) 
		  throw new Error (`Schema must be ${this.validSchemas.join(', ')}`);


		 if (!valid) {
		   throw new Error(ajv.errorsText(validate.errors))
```